tests/unit/test_extensions/test_litestar/test_init_plugin/test_asyncio.py

```diff
@@ -309,3 +323,92 @@ async def test_autocommit_handler_maker_multi(create_scope: Callable[..., Scope]
     await config2.before_send_handler(http_response_start, http_scope)  # type: ignore
     mock_session2.rollback.assert_called_once()
     mock_session1.rollback.assert_not_called()
+
+
+@pytest.mark.parametrize(
+    ("exc", "status"),
+    [
+        (IntegrityError, HTTP_409_CONFLICT),
+        (ForeignKeyError, HTTP_409_CONFLICT),
+        (DuplicateKeyError, HTTP_409_CONFLICT),
+        (InvalidRequestError, HTTP_500_INTERNAL_SERVER_ERROR),
+        (NotFoundError, HTTP_404_NOT_FOUND),
+    ],
+)
+def test_repository_exception_to_http_response(exc: type[RepositoryError], status: int) -> None:
+    """Test default exception handler."""
+
+    config1 = SQLAlchemyAsyncConfig(connection_string="sqlite+aiosqlite://")
+    config2 = SQLAlchemyAsyncConfig(
+        connection_string="sqlite+aiosqlite://",
+        session_dependency_key="other_session",
+        session_scope_key="_sqlalchemy_state_2",
+        engine_dependency_key="other_engine",
+    )
+    plugin = SQLAlchemyInitPlugin(config=[config1, config2])
```

Comment:
Is there a reason this test needs 2 configs? Removing one would keep the test simpler IMO